migration/migration.go

```diff
@@ -16,14 +17,23 @@ import (
 	"github.com/juju/version"
 	"gopkg.in/juju/charm.v6"
 
+	"github.com/juju/juju/core/leadership"
 	"github.com/juju/juju/core/migration"
+	"github.com/juju/juju/feature"
 	"github.com/juju/juju/resource"
 	"github.com/juju/juju/state"
 	"github.com/juju/juju/tools"
 )
 
 var logger = loggo.GetLogger("juju.migration")
 
+// When we import a new model, we need to give the leaders some time to
+// settle. We don't want to have leader switches just because we migrated an
+// model, so this time needs to be long enough to make sure we cover
+// the time taken to migration a reasonable sized model. We don't yet
```

Comment:
s/migration/migrate

Reply:
Yup - I just cut-n-pasted this without reading it closely enough, I'm blaming @mjs for the original typos.

Reply:
I apologise for any migraines my migratory migrations typos may have caused :/